src/odemis/gui/cont/tabs.py

```diff
@@ -1706,6 +1706,101 @@ def get_display_priority(cls, main_data):
             return None
 
 
+class FastEMOverviewTab(Tab):
+    def __init__(self, name, button, panel, main_frame, main_data):
+
+        # During creation, the following controllers are created:
+        #
+        # ViewPortController
+        #   Processes given viewport.
+        #
+        # StreamController
+        #   Manages the single beam stream.
+        #
+        # Acquisition Controller
+        #   Takes care of the acquisition and acquisition selection buttons.
+
+        tab_data = guimod.FastEMOverviewGUIData(main_data)
+
+        super(FastEMOverviewTab, self).__init__(name, button, panel, main_frame, tab_data)
+        self.set_label("OVERVIEW")
+
+        # View Controller
+        vp = panel.vp_fastem_overview
+        assert(isinstance(vp, FastEMOverviewViewport))
+        vpv = collections.OrderedDict([
+            (vp,
+             {"name": "Overview",
+              "stage": main_data.stage,
+              "cls": guimod.StreamView,
+              "stream_classes": EMStream,
+              "zPos": self.tab_data_model.zPos,
+              }),
+        ])
+        self.view_controller = viewcont.ViewPortController(tab_data, panel, vpv)
+
+        # Single-beam SEM stream
+        vanames = ("resolution", "dwellTime", "horizontalFoV", "pixelSize")
```

Comment:
BTW, you might want to double check it still works fine with the latest update on xt_client, now that resolution is a read-only EnumeratedVA.
If you want to allow the user to change the resolution, I think you'd need a special control, which shows the resolution, but actually changes the scale.

Reply:
Could we just use the normal scale control instead and display the resolution as read-only?

Reply:
I think that scale + resolution read-only would be completely fine. Of course, in the longer run we might want to have something more polished...